src/components/sprite-selector/sprite-selector.jsx

```diff
@@ -127,6 +128,10 @@ const SpriteSelectorComponent = function (props) {
                         title: intl.formatMessage(messages.addSpriteFromPaint),
                         img: paintIcon,
                         onClick: onPaintSpriteClick // TODO need real function for this
```

Comment:
Do you know what these TODOs mean and if they are still relevant?

Reply:
I don't!